### PR TITLE
Fix custom factor math...

### PR DESCRIPTION
**Pull request by dellaert**
... and 99.9996 percentage

### CODE REVIEW

**Comment by dellaert** [comment]
Fixes #48

**Comment by dellaert** [comment]
@gchenfc pls approve and merge unless you see issues.

File: _tutorials/intro.html [hunk 1]
```diff
@@ -2043,7 +2046,7 @@ <h3 class="subsection" id='magicparlabel-65550'><span class="subsection_label">4
...
+<div class="standard" id='magicparlabel-65581'>We can optimize this factor graph as before, by creating an initial estimate of type <em><b>Values</b></em>, and creating and running an optimizer. The result is shown graphically in Figure <a href="#fig_example">7</a>, along with covariance ellipses shown in green. These covariance ellipses in 2D indicate the marginal over position, over all possible orientations, and show the area which contain 99.9996% of the probability mass. The graph shows in a clear manner that the uncertainty on pose <math xmlns="http://www.w3.org/1998/Math/MathML">
```
Comment (gchenfc) [inline _tutorials/intro.html:2049]: These **3-sigma** covariance ellipses in 2D indicate the marginal over position, over all possible orientations, and show the area which contain 99.9996% of the probability mass.

**Review by gchenfc — CHANGES_REQUESTED** [review]
See https://github.com/borglab/gtsam/pull/1064

**Review by gchenfc — APPROVED** [review]

**Pull request merged**